cmd/thor/utils.go

```diff
@@ -416,14 +418,7 @@ func loadNodeMaster(ctx *cli.Context) (*node.Master, error) {
 	return master, nil
 }
 
-type p2pComm struct {
-	comm           *comm.Communicator
-	p2pSrv         *p2psrv.Server
-	peersCachePath string
-	enode          string
-}
-
-func newP2PComm(ctx *cli.Context, repo *chain.Repository, txPool *txpool.TxPool, instanceDir string) (*p2pComm, error) {
+func newThorP2P(ctx *cli.Context, repo *chain.Repository, txPool *txpool.TxPool, instanceDir string) (*p2p.ThorP2P, error) {
```

Comment:
`ThorP2P` sounds strange to me, what do you think about the old one `P2PComm`

Reply:
That kinda already exists right? Well not exactly p2pcomm but there is a comm package.. I found these confusing, to be honest that's why I try to split them.
I chose ThorP2P because it's the package that initiates all of thor's P2P needs (flags, discovery and comms). 
Not attached to it though, perhaps there another name that could work?